cfgov/jinja2/v1/_includes/molecules/text-introduction.html

```diff
@@ -29,10 +31,19 @@
                         at the bottom of the molecule.
 
    ========================================================================== #}
-
+{% if value.eyebrow and value.heading %}
+<header>
```

Comment:
This `header` element doesn't seem to be serving any purpose. Removing it on a rendered page with dev tools shows no difference.

Can we remove it? We also would not need the two conditionals, then.

Reply:
I'd added the `header` to associate the eyebrow with the H1, but you're right, it doesn't seem to make a difference. Removed in https://github.com/cfpb/cfgov-refresh/pull/4004/commits/357720b548aa58d499fd2d6513fa891e82f8f79b